.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,31 +6,33 @@ labels: bug
 assignees: ''
 ---
 
-**Describe the bug**
+## Describe the bug
```

Comment:
You can enforce using template upon the authors if you put `.github/ISSUE_TEMPLATE/config.yml` and set `blank_issues_enabled: false`.

ref: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

If you need it, I can add it.